arch/arm/src/cxd56xx/cxd56_rtc.c

```diff
@@ -254,13 +254,8 @@ static void cxd56_rtc_initialize(int argc, ...)
 {
   struct timespec ts;
 #ifdef CONFIG_CXD56_RTC_LATEINIT
-  static WDOG_ID s_wdog = NULL;
-  static int     s_retry = 0;
-
-  if (s_wdog == NULL)
-    {
-      s_wdog = wd_create();
-    }
+  static struct wdog_s s_wdog;
```

Comment:
@masayuki2009 Is there a problem with this?  A single static instance implies that there will be only a single instance of the cxd_rtc_initialize() active.  Is this a valid assumption?  This is the only case like this I have seen in the review.

Reply:
Yes, this is the only case which use a static/global variable, all other place put WDOG_ID in some type of structure.